### PR TITLE
fix stripttc.bundle for gems with prebuilt extension

### PR DESCRIPTION
**Pull request by CAMOBAP**
- https://github.com/fontist/fontist/pull/313#issuecomment-1426952532

### CODE REVIEW

**Comment by CAMOBAP** [comment]
Once approved I will release new version to unblock https://github.com/fontist/fontist/issues/308

**Review by maxirmx — APPROVED** [review]

**Pull request merged**
